### ISSUE TITLE
Thread.Sleep causes steps to fail after 30s

### ISSUE THREAD

**Issue by fruityoat** [getgauge/gauge#905]
### Expected behavior
Step should pass after the allotted time, as it did before 0.9.6

### Actual behavior
Step fails if the thread is slept for time is 30s or more. Step passes if it is 29 seconds or less.

>  Failed Step: Wait for "30" seconds
>  Specification: specs\scratch.spec:7
> Error Message: Request timedout for Message ID => _xxx_
> Stacktrace:
>

### Steps to reproduce
Run this step from a spec with seconds >= 30
```
[Step("Wait for <seconds> seconds")]
public void WaitSeconds(int seconds)
{
    Thread.Sleep(1000 * seconds);
}
```

### Gauge version
```
Gauge version: 0.9.6
Commit Hash: 571fda8

Plugins
-------
csharp (0.10.1)
html-report (4.0.2)
screenshot (0.0.1)
xml-report (0.2.0)
```

**Comment by nehashri** [comment]
@fruityoat This happens because Gauge doesn't understand depend on Java code. It is the java runner's thread that is put to sleep, but Gauge waits for a response from Java and times out. You can avoid this by setting an appropriate request timeout in for gauge. You can set [this](https://docs.getgauge.io/configuration.html#gauge-properties-28) configuration in gauge.properties file. Or run the command 
```
gauge config runner_request_timeout <timeout in miliseconds>
```

**Comment by fruityoat** [comment]
Thanks that works. Just curious what changed from 0.9.5 to cause this? The change broke our existing tests and it wasn't apparent from the change log that it was intentional.

Also I tried sleeping for 1 second at a time and performing an action in between but that also times out.

**Comment by nehashri** [comment]
@fruityoat we did not change anything with respect to this. This behaviour was expected to work as such always.

**Comment by fruityoat** [comment]
Then apparently it was accidentally 'fixed' in 0.9.6 because it doesn't time out in 0.9.5 ;)

**Comment by zabil** [comment]
Reverted to old behavior in gauge release 0.9.7
